src/comp/comp.py

```diff
@@ -1,5 +1,8 @@
-# The following list comprehension exercises will make use of the 
-# defined Human class. 
+# The following list comprehension exercises will make use of the
+# defined Human class.
+import math
+
+
 class Human:
```

Comment:
Great job on this implementation and for getting the associated test to pass, @MelvinPeepers! 👍 
<img width="626" alt="melvin_comp" src="https://user-images.githubusercontent.com/16110931/76147873-b7877b80-606e-11ea-808b-16a113fbd04f.png">